### DISCUSSION TITLE
nFeature_RNA count vs DoubletFinder for filtering out putative Doublets in sample

### DISCUSSION THREAD

**Discussion by afuiten** [satijalab/seurat#6550] [Q&A]
Hello, 
 
With Seurat, I quality filtered cells based on number of unique genes detected in each cell (nFeature_RNA) – along with percent mitochondrial and ribosomal reads.  I read that the filtering out of cells with a high number of unique genes detected should filter out putative cell doublets or multiplets. 
 
Shouldn't this step alone take care of removing putative doublets in my sample? Why do other people use additional doublet filtering programs like DoubletFinder? Shouldn’t this quality filtering step in Seurat with nFeature_RNA take care of removing the putative doublets?
 
If you could offer me any insight as to why I might need to consider doing anything in addition to removing putative doublets, that would be greatly appreciated. Thank you!

**Comment by lucygarner** [comment]
This depends on the type of sample you have. If you have a mixed population, then the expected number of genes/cell will vary depending on cell type. Therefore, it might not be sensible to filter on the number of genes/cell. The same is true if you have a mix of activated and non-activated cells. Filtering on the number of genes/cell will also not catch all doublets.

DoubletFinder and similar algorithms identify doublets based on mixed transcriptional profiles. So they are able to identify doublets comprising e.g. a T cell and a monocyte, but not doublets comprising e.g. two T cells. Doublets might not necessarily have the highest genes/cell, again depending on the type of cells present in your sample.
